Resources/Storage/test/Test_StorageElement.py

```diff
@@ -81,6 +81,79 @@ def mock_StorageFactory_generateStorageObject( storageName, pluginName, paramete
 
   return S_OK( storageObj )
 
+def mock_StorageFactory_getConfigStorageName( storageName, referenceType, rootConfigPath ):
```

Comment:
There's no difference between this one and the one in Test_FilePlugin.py, right?